apps/avatax/e2e/graphql/mutations/CreateCheckoutWithTwoItems.graphql

```diff
@@ -0,0 +1,28 @@
+mutation CreateCheckoutWithTwoLines(
+  $channelSlug: String!
+  $variantOneId: ID!
```

Comment:
**Question:** Can we put lines here and make this mutation `CreateCheckout`?

I'm thinking if having `lines` as a argument to mutation won't help us in the long run - if I another test I want to add third line now I have to change mutation name, add another argument 🤔